atmo/jobs/urls.py

```diff
@@ -14,4 +14,5 @@
     url(r'^(?P<id>\d+)/edit/', views.edit_spark_job, name='jobs-edit'),
     url(r'^(?P<id>\d+)/run/', views.run_spark_job, name='jobs-run'),
     url(r'^(?P<id>\d+)/$', views.detail_spark_job, name='jobs-detail'),
```

Comment:
Do you think we should rename the previous view to match? Like `detail_jupyter_job`?